README.md

```diff
@@ -18,11 +18,11 @@
 pnpm add -D eslint @antfu/eslint-config
 ```
 
-### Config `.eslintrc`
+### Config `.eslintrc.js`
 
-```json
-{
-  "extends": "@antfu"
+```js
+module.exports = {
+  extends: '@antfu'
 }
```

Comment:
I would prefer to keep it as is. People who need this should probably learn it from ESLint's doc.

Reply:
Sounds you are right